_sass/custom/custom.scss

```diff
@@ -1404,8 +1404,7 @@ main {
 version-selector {
   z-index: 1;
   font-size: .9rem;
-  padding-top: 1rem;
-  padding-bottom: 1rem;
```

Comment:
The padding was adding a space between the version-selector's trigger and the list. I converted it to a margin.